src/test/java/edu/oakland/test/database/DatabaseGisInterfaceTest.java

```diff
@@ -0,0 +1,58 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisInterfaceImplementation;
+import edu.oakland.production.database.DatabaseGisManagerStub;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database GisInterface Test")
+public class DatabaseGisInterfaceTest {
+
+  @Test
+  @DisplayName("LDP going in the same LDP coming out")
+  void dataPointInIsDataPointOut() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    LocationDataPoint ldp = new LocationDataPoint(0, 0, LocalDateTime.of(
+        (int) (Math.random() * 50 + 1970),
+        (int) (Math.random() * 12 + 1),
+        (int) (Math.random() * 28 + 1),
+        (int) (Math.random() * 24),
+        (int) (Math.random() * 60)
+    ));
+    int i = 0;
+    dgi.receiveStoreRequest(ldp);
+    assertEquals(null, dgms.getLocationDataPoint(i));
+  }
+
+  @Test
+  @DisplayName("Putting in the current satellite returns the new satellite")
+  void currentSatGetsNextSat() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    String currentSat = "";
+    assertEquals("", dgi.receiveNextSatRequest(currentSat));
```

Comment:
To actually verify that everything is being passed directly, can you place a test string there, and then reference `currentSat` as the expected output?